configure.ac

```diff
@@ -416,6 +416,7 @@ dnl are usually in libnsl
 dnl Also, uClibc will bark at linking with glibc's libnsl.
 
 PHP_CHECK_FUNC(socket, socket)
+AC_CHECK_LIB(network, socket)
```

Comment:
You could amend the preceding PHP_CHECK_FUNC() instead.
````
PHP_CHECK_FUNC(socket, socket, network)
````

Reply:
Ok, i'll fix this.